src/schema/nodeNames.js

```diff
@@ -0,0 +1,9 @@
+export const NodeNames = {
+  TABLE: 'table',
+  TABLE_ROW: 'table_row',
+  TABLE_CELL: 'table_cell',
+  TABLE_HEADER: 'table_header',
+  CHECKBOX: 'checkbox',
+  DATE: 'date',
+  LABEL: 'label',
```

Comment:
nit - trailing comma?

Reply:
We had trailing commas turned on in prettierrc, I have switched it to `none` and ran prettier on the repo